room.py

```diff
@@ -0,0 +1,90 @@
+from enum import Enum
+from deck import Deck
+
+
+class room:
+    class players(Enum):
+        p1 = 1
+        p2 = 2
+        p3 = 3
+        p4 = 4
+
+    class Teams(Enum):
+        Blue = 1
+        Red = 2
+
+    class bid_status(Enum):
+        Pass = 1
+        Double = 2
+        Bid = 3
+
+    def __init__(self, user1, user2, user3, user4):
+        self.player = {
+            self.players.p1: user1,
+            self.players.p2: user2,
+            self.players.p3: user3,
+            self.players.p4: user4
+        }
+        self.highest_bidder = None
+        self.current_bidder = None
+        self.trump_card = None
+        self.current_trump = None
+        self.contract = None
```

Comment:
Do we really need to have both contract and current_contract properties for the room ?